### PR TITLE
Nicely style past event cards

### PR DESCRIPTION
**Pull request by Hoverbear**

### CODE REVIEW

**Comment by dns2utf8** [comment]
Wow, it is beautiful. But it issues a reload of all the background picture with every resize on Chromium.

File: _sass/_card.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    }
+}
+
+// Cards in general
+
+.cardset .card {
+    padding: 2vw;
+    background-image: 
+        linear-gradient(to bottom, rgba(8,71,136,0.9), rgba(8,71,136,0.1)),
+        var(--banner);
```
Comment (dns2utf8) [inline _sass/_card.scss]: magic

File: _includes/cards/event.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<div id="{{event.slug}}" class="card event" style="--banner: url('/assets/past_events/{{ event.slug }}.png')">
```
Comment (dns2utf8) [inline _includes/cards/event.html:1]: magic

**Review by dns2utf8 — COMMENTED** [review]
This `--banner` is new / dark magic. Searching for docs

File: _includes/cards/event.html (hunk 2 above)
Comment (Hoverbear) [reply _includes/cards/event.html:1]: Welcome to CSS vars =D https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables

**Pull request merged**